code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -3577,7 +3577,17 @@
 				if(prob(10))
 					to_chat(H,"<span class='warning'>You really don't feel very good.</span>")
 				if(prob(5))
-					H.adjustToxLoss(0.1)
+					switch(pick(1,2,3,4,5))
```

Comment:
I noted in one place that he quintupled the damage, but I didn't point out that in another, he nearly *octupled* it, with a 1/5 chance of quadrupling *that*

I'm treating it as a meme PR, at least, unless he says something to justify it